ntbs-service/DataAccess/DataQualityRepository.cs

```diff
@@ -106,24 +106,31 @@ public async Task<IList<Notification>> GetNotificationsEligibleForDotVotAlerts()
 
         public async Task<IList<NotificationAndDuplicateIds>> GetNotificationIdsEligibleForPotentialDuplicateAlerts()
         {
-            return await GetBaseNotificationQueryableForAlerts()
-                .SelectMany(a => _context.Notification, 
+            return await _context.Notification
+                .SelectMany(_ => _context.Notification, 
                     (notification, duplicate) => new {notification, duplicate})
                 .Where(t =>
+                    // Check that one of the following cases are true:
+                    // The notification's names match directly (and it is not matching on itself)
+                    (t.notification.PatientDetails.GivenName == t.duplicate.PatientDetails.GivenName &&
+                     t.notification.PatientDetails.FamilyName == t.duplicate.PatientDetails.FamilyName &&
+                     t.notification.NotificationId != t.duplicate.NotificationId)
```

Comment:
this feels like one of the general conditions we should impose at the bottom (in fact, I think the DOB match case could be suffering from this not being applied right now!)

Reply:
Good point, this crossed my mind but you've made an excellent case for it